keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning/README.md

```diff
@@ -0,0 +1,1042 @@
+	if pl.scorer == nil {
+		return 0, nil
+	}
+	state, err := getStateData(cs)
+	if err != nil {
+		return 0, framework.AsStatus(err)
+	}
+	podVolumes, ok := state.podVolumesByNode[nodeName]
+        if !ok {
+		return 0, nil
+	}
+-       // group by storage class
++
+        classResources := make(classResourceMap)
+-       for _, staticBinding := range podVolumes.StaticBindings {
+-               class := staticBinding.StorageClassName()
+-               storageResource := staticBinding.StorageResource()
+-               if _, ok := classResources[class]; !ok {
+-                       classResources[class] = &StorageResource{
+-                               Requested: 0,
+-                               Capacity:  0,
++       if len(podVolumes.StaticBindings) != 0 {
++               // group static biding volumes by storage class
++               for _, staticBinding := range podVolumes.StaticBindings {
++                       class := staticBinding.StorageClassName()
++                       storageResource := staticBinding.StorageResource()
++                       if _, ok := classResources[class]; !ok {
++                               classResources[class] = &StorageResource{
++                                       Requested: 0,
++                                       Capacity:  0,
++                               }
++                       }
++                       classResources[class].Requested += storageResource.Requested
++                       classResources[class].Capacity += storageResource.Capacity
++               }
++       } else {
++               // group dynamic biding volumes by storage class
++               for _, provision := range podVolumes.DynamicProvisions {
++                       if provision.Capacity == nil {
++                               continue
++                       }
++                       class := *provision.PVC.Spec.StorageClassName
++                       if _, ok := classResources[class]; !ok {
++                               classResources[class] = &StorageResource{
++                                       Requested: 0,
++                                       Capacity:  0,
++                               }
+                        }
++                       requestedQty := provision.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
++                       classResources[class].Requested += requestedQty.Value()
++                       classResources[class].Capacity += provision.Capacity.Capacity.Value()
+                }
+-               classResources[class].Requested += storageResource.Requested
+-               classResources[class].Capacity += storageResource.Capacity
+        }
++
+        return pl.scorer(classResources), nil
+}
+```
+
+Users can select the scoring logic from the following options in `VolumeBindingargs`. The scoring logic is the same among all Pod + PVC(s).
```

Comment:
the "args" part isn't capitalized for some reason: `VolumeBindingArgs`

Reply:
I addressed it.